google/ggrc-core
# GGRC-6078 Fix review status change when applying proposal for ACL role

# Issue description

Currently when applying proposal which contains only ACL role changes to a reviewed control, control becomes unreviewed but it should not.

# Steps to test the changes

1. Create a control;
2. Mark it as reviewed;
3. Click "Propose changes" button;
4. Add admin or primary/secondary contact, click "Save";
5. Apply proposal in change proposal tab;
6. Open attributes tab and look at the review status;
7. Review status should stay "reviewed".

# Solution description

Make method **"handle_proposal_applied"** that handles **"proposal_applied"** signals to call **"_update_status_on_attrs"** and **"_update_status_on_custom_attrs"** which updates review status based on attributes has been changed.

Also add attribute **"modified_by"** to **"ATTRS_TO_IGNORE"** set which specifies attributes whose changes does not invoke review status change.

# Sanity checklist




# PR Review checklist
